packages/cli/config/jest.js

```diff
@@ -48,7 +48,7 @@ function getRoleConfig(role) {
     case 'common-library':
     case 'frontend-plugin':
     case 'frontend-plugin-module':
-      return { testEnvironment: require.resolve('jest-environment-jsdom') };
+      return { testEnvironment: require.resolve('./jsdomEnvironment') };
```

Comment:
everything else in the config dir is namespaced, could we make this `jestJsdomEnvironment`?